### PR TITLE
Model: Callisto param update

### PR DESCRIPTION
**Pull request by lthall**
Update of the Callisto Model parameters to add a lidar and improve yaw behaviour.

A note for developers, the added D term is required because we do not include inertia in our motor/propeller model.

### CODE REVIEW

File: Tools/autotest/models/Callisto.param [hunk 1]
```diff
@@ -46,14 +47,22 @@ MOT_THST_HOVER,0.52
...
+PSC_ACCZ_IMAX,800
+PSC_ACCZ_P,0.25
 PSC_ANGLE_MAX,45
 PSC_JERK_Z,10
 PSC_POSZ_P,0.5
 PSC_VELZ_P,2.5
+RNGFND1_MAX_CM,10000
+RNGFND1_ORIENT,25
+RNGFND1_PIN,0
+RNGFND1_SCALING,12.1212
```
Comment (rmackay9) [inline Tools/autotest/models/Callisto.param:64]: I think some of these RNGFND1_ values are the defaults so we could remove them.  In particular I think these can be removed:

- RNGFND1_ORIENT,25
- RNGFND1_PIN,0
- RNGFND1_SCALING,12.1212

Comment (lthall) [reply Tools/autotest/models/Callisto.param:64]: Confirmed I need:
RNGFND1_PIN,0
RNGFND1_SCALING,12.1212
But I don't need: 
RNGFND1_ORIENT,25
image

Comment (rishabsingh3003) [reply Tools/autotest/models/Callisto.param:64]: RNGFND1_PIN defaults to -1 I think, so setting this to 0 would be better

**Comment by rmackay9** [comment]
Merged, thanks!

**Pull request merged**